py3status/module.py

```diff
@@ -102,7 +107,17 @@ def load_methods(self, module, user_modules):
             # apply module configuration from i3status config
             mod_config = self.i3status_thread.config.get(module, {})
             for config, value in mod_config.items():
-                setattr(self.module_class, config, value)
+                # names starting with '.' are private
```

Comment:
The reason I've used `.` is to keep the namespace clean.  It is possible that some user module uses a `_variable` in it's config and I didn't want to break such modules.

I can change to the underscore/double if you prefer but that was my reasoning.